language/stdlib/modules/LibraAccount.move

```diff
@@ -151,7 +151,7 @@ module LibraAccount {
         if (is_withdrawal) {
             VASP::is_vasp(payer) && (!VASP::is_vasp(payee) || !VASP::is_same_vasp(payer, payee))
         } else {
-            VASP::is_vasp(payee) && (!VASP::is_vasp(payee) || !VASP::is_same_vasp(payer, payee))
+            VASP::is_vasp(payee) && (!VASP::is_vasp(payer) || !VASP::is_same_vasp(payee, payer))
```

Comment:
@sblackshear @tzakian I think this was a bug in the code. Discovered while trying to verify things.

Reply:
Nice!!